compose/service.py

```diff
@@ -399,6 +401,12 @@ def _get_container_create_options(self, override_options, one_off=False):
             self.containers(stopped=True, one_off=one_off),
             one_off)
 
+        if 'labels' in container_options:
+            labels_dict = ['io.docker.compose:' + label for label in container_options['labels']]
```

Comment:
Ideally, labels used for scheduling (Swarm) should be namespaced and perhaps not even defined as *labels* in `docker-compose.yml` (even though they will be set as labels), e.g.

```yaml
web:
    constraints:
        storage: ssd
        memory: 2g
    labels:
        hello: world
```

the "constraints" will then result in a `--label com.docker.swarm.constraint=storage=ssd` (or similar)

I think Swarm is planning on namespacing their labels as well, but I'll have to check.